app/models/product.rb

```diff
@@ -22,6 +24,11 @@ class Product < ApplicationRecord
   validates_comparison_of :words_in_permalink_separated_by_hyphen, greater_than_or_equal_to: 3, message: "permalink should contain minimun 3 words separated by hyphen", if: :permalink_present?
   validates_comparison_of :words_in_description, greater_than_or_equal_to: 5, less_than_or_equal_to: 10, if: ->{ description.present? }
 
+  scope :enabled_and_price_above, ->(price) { where("enabled = ? AND price > ?", true, price) }
+  scope :products_in_cart, -> { Product.joins(:line_items) }
```

Comment:
dont add scopes for the queries. just add the queries as comments on the PR

Reply:
will this give me repeated products? please check

Reply:
Should i remove enabled_and_price_above scope?

Reply:
Yes, :products_in_cart will give repeated products, fixed the issue

Reply:
yes remove the scope and write the queries as you would run in  a console as a comment on this PR